.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: A short description of the new feature
+labels: enhancement
+assignees: sinukaarel
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```

Comment:
Screenshots section overlaps with Additional context. I would perhaps even remove the Screenshots section, as it might seem it is mandatory for the feature request.

Reply:
Removed screenshots section https://github.com/sendsmaily/smaily-opencart-module/pull/80/commits/df54c1e23ef01dafd971b5b6e948f1dca2b3c9f3